src/plugins/intel_gna/src/serial/gna_model_serial.hpp

```diff
@@ -12,8 +12,11 @@
 #include "descriptions/gna_desc.hpp"
 #include "gna2-model-api.h"
 #include "gna_device_allocation.hpp"
+#include "pre_post_process/transposition_info.hpp"
 #include "serial/headers/latest/gna_model_header.hpp"
 
+using namespace ov::intel_gna::pre_post_processing;
```

Comment:
please don't use using namespace in header

Reply:
fixed